test/tests/parallel/set1/spam_virus/fail2ban.bats

```diff
@@ -73,8 +73,8 @@ function teardown_file() {
 @test "ban ip on multiple failed login" {
   CONTAINER1_IP=$(_get_container_ip "${CONTAINER1_NAME}")
   # Trigger a ban by failing to login twice:
-  CONTAINER_NAME=${CONTAINER2_NAME} _send_email 'auth/smtp-auth-login-wrong' "${CONTAINER1_IP} 465"
-  CONTAINER_NAME=${CONTAINER2_NAME} _send_email 'auth/smtp-auth-login-wrong' "${CONTAINER1_IP} 465"
+  CONTAINER_NAME=${CONTAINER2_NAME} _nc_wrapper 'auth/smtp-auth-login-wrong.txt' "${CONTAINER1_IP} 465"
+  CONTAINER_NAME=${CONTAINER2_NAME} _nc_wrapper 'auth/smtp-auth-login-wrong.txt' "${CONTAINER1_IP} 465"
```

Comment:
This would presumably resolve https://github.com/docker-mailserver/docker-mailserver/issues/3713

Reply:
This alone won't, but I will take care of that by using `_send_email` instead of `nc`.